src/theme.js

```diff
@@ -0,0 +1,17 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+const theme = createMuiTheme({
+  icon: {
+    fontSize: '40px',
+    top: '2px',
```

Comment:
what do you have this for? it is not doing anything

Reply:
the whole theme? it is used in both language selector and footer for the icons.